TEMPLATE/etc/httpd/conf.d/manageiq-external-auth.conf.erb

```diff
@@ -37,7 +37,7 @@ LoadModule auth_kerb_module modules/mod_auth_kerb.so
 </LocationMatch>
 
 
-<LocationMatch ^/api|^/vmdbws/wsdl|^/vmdbws/api>
+<LocationMatch ^/api(?!\/product_info$)|^/vmdbws/wsdl|^/vmdbws/api>
```

Comment:
this should match the optional versioned path, i.e. /api/v4.1.0/product_info, as per https://github.com/ManageIQ/manageiq-api/blob/086814a4a80cd1e83e2125338f96859d94394b20/lib/api.rb#L3, though probably not used by the SUI, but for completeness if anything.

At some point, we should also remove the /vmdbws/* entries, (the SOAP interface), those are LONG gone.

Reply:
OK, removing vmdbws.

And looks like `^\/api(?!\/(v[\d\.]+\/)?product_info$)` is the right negative lookahead to handle an optional `v + numbers and dots`

Reply:
(@abellotti That's the only format for the api versions, right? `v1` `v1.2.3` etc.)